src/main/scala/akka/stream/alpakka/xlsx/StyleStreamer.scala

```diff
@@ -1,39 +1,59 @@
 package akka.stream.alpakka.xlsx
 
-import java.io.InputStream
 import java.util.zip.ZipFile
 
 import akka.stream.Materializer
-import akka.stream.alpakka.xml.{ EndElement, ParseEvent, StartElement }
 import akka.stream.alpakka.xml.scaladsl.XmlParsing
-import akka.stream.scaladsl.{ Keep, Sink, Source, StreamConverters }
+import akka.stream.alpakka.xml.{EndElement, ParseEvent, StartElement}
+import akka.stream.contrib.ZipInputStreamSource.ZipEntryData
+import akka.stream.scaladsl.{Keep, Sink, Source, StreamConverters}
+import akka.util.ByteString
 
 import scala.collection.mutable
 import scala.concurrent.Future
 import scala.util.Try
 
 object StyleStreamer {
 
-  def readStyles(file: ZipFile)(implicit materializer: Materializer): Future[Map[Int, Int]] = {
-    readStyles(file, Sink.fold(Map.empty[Int, Int])((v1, v2) => v1 + v2))
+  private final val EntryName = "xl/styles.xml"
+  private val defaultSink = Sink.fold[Map[Int, Int], (Int, Int)](Map.empty)((v1, v2) => v1 + v2)
+
+
+  def readStyles(zipFile: ZipFile)(implicit materializer: Materializer): Future[Map[Int, Int]] = {
+    readStyles(zipFile, defaultSink)
   }
 
   def readStyles(
-      file: ZipFile,
+      zipFile: ZipFile,
```

Comment:
This change is fine here, but it would not be source compatible for the "public" version of the library (it breaks on named parameters).

Reply:
Should we care about it now? It doesn't matter for us probably (we aren't going to use `ZipFile`-related API at all) and the `akka-xlsx` author'd decide whether this change is appropriate when merging this fork.

Reply:
Just sayin